boardgame-ui/src/pages/TimerCreationPage.vue

```diff
@@ -1,265 +1,36 @@
-
-            <b-field v-if="timer.timer_type === 'RELOAD'" :label="$t('timer.add-edit.timer.reload-increment')">
-              <b-numberinput min="0" controls-position="compact" v-model="timer.reload_increment"/>
-            </b-field>
-          </b-field>
-
-          <b-field :label="$t('add-edit-game.board-game.label')">
-            <b-autocomplete
-              v-model="searchString"
-              :data="filteredBoardGames"
-              field="name"
-              icon="search"
-              @select="selectBoardGame"
-              name="boardGame"
-              :data-vv-as="$t('add-edit-game.board-game.label')"
-            >
-              <template slot="empty">{{$t('add-edit-game.board-game.no-result')}}</template>
-            </b-autocomplete>
-          </b-field>
-
-          <h2 class="subtitle">{{$t('add-edit-game.players.title')}}</h2>
-
-          <table class="table is-fullwidth">
-            <thead>
-            <tr>
-              <th>{{$t('add-edit-game.players.user')}}</th>
-              <th>{{$t('global.color')}}</th>
-              <th class="has-text-white">.</th>
-            </tr>
-            </thead>
-            <tbody>
-            <tr v-for="({user}, idx) in players" :key="user ? `user${user.id}` : idx">
-              <td>
-                <b-field
-                  :type="{'is-danger': errors.has(`user-${idx}`)}"
-                  :message="errors.first(`user-${idx}`)"
-                >
-                  <user-autocomplete
-                    size="is-small"
-                    v-model="players[idx].user"
-                    :users="allUsers"
-                    :excludedIds="selectedUsersIds"
-                    :name="`user-${idx}`"
-                    :data-vv-as="$t('add-edit-game.players.user')"
-                    v-validate="'required'"
-                  />
-                </b-field>
-              </td>
-              <td>
-                <verte
-                  v-model="players[idx].color"
-                  picker="square"
-                  model="hex"
-                  :enableAlpha="false"></verte>
-              </td>
-              <td>
-                <button type="button" class="delete" @click="removePlayer(idx)"></button>
-              </td>
-            </tr>
-            <tr>
-              <td colspan="3" class="has-text-centered">
-                <button class="button is-small" type="button" @click="addPlayer()">
-                  {{$t('button.add-player')}}
-                </button>
-              </td>
-            </tr>
-            </tbody>
-          </table>
-
-          <div class="buttons is-right">
-            <router-link tag="button" class="button is-light" :to="{name: 'timers'}">{{$t('button.cancel')}}</router-link>
-            <button class="button is-primary">{{$t('button.save')}}</button>
-          </div>
-
-        </form>
-
-      </section>
-    </div>
+    <add-timer-form 
+      class="form"
+      @timerCreated="goBackToTimers"
```

Comment:
If editing a timer from the event, maybe better to return to the event? To discuss because if we keep both pages, we may not know from which page the user comes from.

Reply:
This should be the current behavior (if it is not, it is a bug). `TimerCreationPage` is only used for timers outside of events, that's why we go back to the timers list in this case.